RELEASE.md

```diff
@@ -0,0 +1,6 @@
+Release type: patch
```

Comment:
Lets at least make this a minor because technically, we're "breaking" invalid messages silently accepted before.

Reply:
I agree, was thinking about it too.